### PR TITLE
Always set HTML LANG attribute in login templates

### LINKED ISSUE #35103: [LoginUI] Set HTML lang attribute to "en" when internationalization disabled
### Description

There are some issues with the current handling of the `lang` attribute in the html tag within the `template.ftl`:

`<html class="${properties.kcHtmlClass!}"<#if realm.internationalizationEnabled> lang="${locale.currentLanguageTag}"</#if>>`

The `lang` attribute is set only when internationalization is enabled, and it is omitted entirely when internationalization is disabled. This behavior could cause problems for screen readers, which rely on the lang attribute for accurate output.

### Discussion

https://github.com/keycloak/keycloak/discussions/35050

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by andreas-blaettlinger**
Fixes #35103

### CODE REVIEW

**Comment by andreas-blaettlinger** [comment]
@jonkoops can you find me someone who can review this? Will be away for quite some time after this week, thanks in advance! :)

File: services/src/main/java/org/keycloak/forms/login/freemarker/FreeMarkerLoginFormsProvider.java [hunk 1]
```diff
@@ -150,6 +152,7 @@ public FreeMarkerLoginFormsProvider(KeycloakSession session) {
         this.realm = session.getContext().getRealm();
         this.client = session.getContext().getClient();
         this.uriInfo = session.getContext().getUri();
+        this.lang = "en";
```
Comment (jonkoops) [inline services/src/main/java/org/keycloak/forms/login/freemarker/FreeMarkerLoginFormsProvider.java:155]: Not sure if this is the same value, but perhaps we can use the [`Locale.ENGLISH`](https://docs.oracle.com/javase/8/docs/api/java/util/Locale.html#ENGLISH) constant?

File: services/src/main/java/org/keycloak/services/error/KeycloakErrorHandler.java [hunk 2]
```diff
@@ -184,12 +184,15 @@ private static Map<String, Object> initAttributes(KeycloakSession session, Realm
         Map<String, Object> attributes = new HashMap<>();
         Properties messagesBundle = theme.getMessages(locale);
 
+        final var localBean =  new LocaleBean(realm, locale, session.getContext().getUri().getRequestUriBuilder(), messagesBundle);
+        final var lang = realm.isInternationalizationEnabled() ? localBean.getCurrentLanguageTag() : "en";
```
Comment (jonkoops) [inline services/src/main/java/org/keycloak/services/error/KeycloakErrorHandler.java:188]: Same here, is there a constant we can use?

File: services/src/main/java/org/keycloak/services/error/KeycloakErrorHandler.java [hunk 3]
```diff
@@ -184,12 +184,15 @@ private static Map<String, Object> initAttributes(KeycloakSession session, Realm
         Map<String, Object> attributes = new HashMap<>();
         Properties messagesBundle = theme.getMessages(locale);
 
+        final var localBean =  new LocaleBean(realm, locale, session.getContext().getUri().getRequestUriBuilder(), messagesBundle);
```
Comment (jonkoops) [inline services/src/main/java/org/keycloak/services/error/KeycloakErrorHandler.java:187]:
```suggestion
        final var localeBean =  new LocaleBean(realm, locale, session.getContext().getUri().getRequestUriBuilder(), messagesBundle);
```

**Review by jonkoops — COMMENTED** [review]
Largely lgtm, just a couple of nits/questions.

**Review by jonkoops — APPROVED** [review]

**Review by pedroigor — APPROVED** [review]

**Pull request merged**